_i18n/es/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,53 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-## Operating Systems:  Windows, Linux, Mac
+{% include disclaimer.html translated="true" version=page.version %}
+## Sistemas Operativos:  Windows, Linux, Mac
 
-- Retrieve your 25 word @mnemonic-seed that you saved when creating your old Monero @wallet
+- Recupera la semilla mnemónica de 25 palabras que guardaste cuando creaste tu viejo monedero de Monero.
 
-### Account Software:  monero-wallet-cli
+### Software de cuenta: monero-wallet-cli
```

Comment:
I thought that it may sound a bit rough, and omitting it will make it more like a title and not something about specifically some account, like when we say "the account".

Reply:
Dicen "helado de la fresa" o "helado de fresa" en español? 
("profesor de ingles" / "profesor del ingles" etc.)

Reply:
"Helado de fresa". That doesn't mean it doesn't have gender. I remember I erased a past comment, maybe I shouldn't have, but it's ok. The comment was a suggestion changing the organization of the words. It was this:

"Cuenta del software". 

Then I had a conflict with the suggestion I made and did some investigations. Got to "Software de la cuenta" because like I stated, it describes the Software for the account, it's making a description of the software, to finally say: `monero-wallet-cli`. 

But is just a comment. I like to share what goes inside my head while I put on words in a comment.